build/run-e2e-tests.sh

```
@@ -135,8 +135,8 @@ test_content_and_local_cluster_report(){
 	pod=$(oc get pods | grep insights-client | cut -d' ' -f1)
 	oc logs $pod
 	log=$(oc logs $pod) 
-	log_msg=$(echo $log| grep "Creating Request for cluster")
-	if [[ "$log_msg" == *"local-cluster"* ]]; then
+	log_msg=$(echo $log| grep "Creating Insights report request")
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)